### DISCUSSION TITLE
Calibrate moisture sensor?

### DISCUSSION THREAD

**Discussion by drkunze** [rasclatt-dot-com/ESP32-Plant-Waterer-for-ESPHome#5] [General]
I have "sort of" gotten the m5atom with watering system units running. My soil moisture value stays on -22 while immersed in soaked peat moss. Is there some setup to do to calibrate?

Thanks

**Comment by nborson** [comment]
Discussions is probably the best place to ask and answer this question, but see this "issue:" https://github.com/rasclatt-dot-com/ESP32-Plant-Waterer-for-ESPHome/issues/3
